client/actions/planning/ui.ts

```diff
@@ -125,11 +125,23 @@ const loadMore = () => (
 
         return dispatch(planningApis.fetch(params))
             .then((items) => {
+                const lastDayGroupItems = selectors.planning.lastDayGroup(getState());
+
                 if (get(items, 'length', 0) === MAIN.PAGE_SIZE) {
                     dispatch(self.requestPlannings(params));
                 }
-                dispatch(self.addToList(items.map((p) => p._id)));
-                return Promise.resolve(items);
+
+                const result: Array<IPlanningItem> = items;
+
+                if ((lastDayGroupItems?.events ?? []).length > 0) {
+                    // add items from group to stored planning items, so grouping works later
+                    dispatch(planningApis.receivePlannings(lastDayGroupItems.events));
+                    result.push(...lastDayGroupItems.events);
```

Comment:
I wonder if it should rather do `lastDayGroupItems + items`, the new items in theory go after, but it's probably re-sorted later anyhow right?